src/core/user_operation.ml

```diff
@@ -1,12 +1,16 @@
 open Helpers
 open Crypto
 type initial_operation =
-  | Transaction    of {
+  | Transaction          of {
       destination : Key_hash.t;
       amount : Amount.t;
       ticket : Ticket_id.t;
     }
-  | Tezos_withdraw of {
+  | Contract_origination of {
+      to_originate : Smart_contracts.Origination_payload.t;
+      ticket : Ticket_id.t;
```

Comment:
What does this ticket mean? If it's the internal amount that the contract can carry, I would say to remove this as we don't have defined how contracts will interact. And it's definitely not in the scope of this PR.

Reply:
removed as out of scope.